public/app/models/query/Group.js

```diff
@@ -0,0 +1,90 @@
+// Based on http://kindohm.github.io/knockout-query-builder/js/group.js
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout',
+            'models/query/Condition'
+        ],
+        function (_, ko, Condition) {
+            var Group;
+
+            Group = function (queryBuilder, parentGroup) {
+                this.children = ko.observableArray();
+                this.selectedOperator = ko.observable();
+
+                this.operators = ko.pureComputed(function () {
```

Comment:
This is fine to be hardcoded - whichever operators are supported should be supported for all groups.